src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -764,6 +764,32 @@ public void TestRound()
             }
         }
 
+        [Test]
+        public void TestCeil()
+        {
+            using (CultureHelper.SetCulture("en-GB"))
+            {
+                Helper.AssertTemplateResult("2", "{{ 1.234678 | ceil }}");

```

Comment:
Please cover the 4 spec examples at

https://shopify.github.io/liquid/filters/ceil/

Importantly do test the different input type i.e. number vs string containing a number.